### PR TITLE
Remove filterevents

### PR DESCRIPTION
**Pull request by slinkydeveloper**
Signed-off-by: Francesco Guardiani <francescoguard@gmail.com>

Part of https://github.com/knative/eventing/pull/4358

### CODE REVIEW

File: hack/tools.go [hunk 1]
```diff
@@ -4,6 +4,7 @@
 Copyright 2020 The Knative Authors.
 
 Licensed under the Apache License, Version 2.0 (the "License");
+	_ "knative.dev/eventing/test/test_images/filterevents"
```
Comment (pierDipi) [inline hack/tools.go:7]: Can you remove this?

Comment (slinkydeveloper) [reply hack/tools.go]: wut?! fixed :smile:

File: hack/tools.go [hunk 2]
```diff
@@ -1,13 +1,12 @@
 // +build tools
 
 /*
-Copyright 2020 The Knative Authors.
-
+Copyright 2020 The Knative Authors
```
Comment (pierDipi) [inline hack/tools.go:4]:
```suggestion
Copyright 2020 The Knative Authors

```

Comment (slinkydeveloper) [reply hack/tools.go:4]: I literally copy pasted this copyright from another file

**Review by pierDipi — APPROVED** [review]
/lgtm

**Pull request merged**